templates/layouts/_base.html

```diff
@@ -14,6 +14,9 @@
   }
 } %}
 
+{# TODO: Needs to be wrapped in some condition to only include this if the theme is census #}
+{% include 'layouts/configs/_census-theme.html' %}
```

Comment:
Everything is the census theme until we sort out out a way to specify themes again.